src/transforms/grok_parser.rs

```diff
@@ -100,23 +137,18 @@ mod tests {
             "bytes": "4263",
         });
 
-        assert_eq!(
-            expected,
-            serde_json::to_value(&event.as_log().all_fields()).unwrap()
-        );
+        assert_eq!(expected, serde_json::to_value(&event.all_fields()).unwrap());
     }
 
     #[test]
     fn grok_parser_does_nothing_on_no_match() {
-        let event = Event::from(r#"help i'm stuck in an http server"#);
-        let mut parser = GrokParserConfig {
-            pattern: String::from("%{HTTPD_COMMONLOG}"),
-            ..Default::default()
-        }
-        .build()
-        .unwrap();
-
-        let event = parser.transform(event).unwrap().into_log();
+        let event = parse_log(
+            r#"help i'm stuck in an http server"#,
```

Comment:
😂 this is good.

Reply:
That's just a mechanical conversion from the existing test. Thank @lukesteensen for the humor. :)